libs/perun/components/src/lib/members-dynamic-list/members-dynamic-list.component.ts

```diff
@@ -127,9 +134,20 @@ export class MembersDynamicListComponent implements AfterViewInit, OnInit, OnCha
       this.groupId,
       this.selectedGroupStatuses
     );
+
+    // Checks if this is 'members' group
+    if (this.groupId) {
+      this.groupManager.getGroupById(this.groupId).subscribe((group) => {
+        this.isMembersGroup = group.name === 'members';
+      }, () => {
+        this.isMembersGroup = false;
```

Comment:
I would initiate isMembersGroup as false and then only set it to true to reduce the amount of confusing code here. (Pretty sure this line shouldn't ever be reached unless an error occurred, maybe atleast add a comment?)